src/algorithms/tracking/IterativeVertexFinder.cc

```diff
@@ -86,11 +86,15 @@ std::unique_ptr<edm4eic::VertexCollection> eicrecon::IterativeVertexFinder::prod
   VertexSeeder::Config seederCfg(ipEst);
   VertexSeeder seeder(seederCfg);
   // Set up the actual vertex finder
-  VertexFinder::Config finderCfg(vertexFitter, std::move(linearizer),
-                                 std::move(seeder), ipEst);
+  VertexFinder::Config finderCfg(std::move(vertexFitter), std::move(linearizer),
+                                 std::move(seeder), std::move(ipEst));
   finderCfg.maxVertices                 = m_cfg.m_maxVertices;
   finderCfg.reassignTracksAfterFirstFit = m_cfg.m_reassignTracksAfterFirstFit;
+  #if Acts_VERSION_MAJOR >= 31
+  VertexFinder finder(std::move(finderCfg));
```

Comment:
I also don't understand why we need an `std::move` here:
https://github.com/acts-project/acts/blob/v30.3.2/Core/include/Acts/Vertexing/IterativeVertexFinder.hpp#L145
takes struct `Config &cfg`
https://github.com/acts-project/acts/blob/v31.2.0/Core/include/Acts/Vertexing/IterativeVertexFinder.hpp#L145
takes struct `Config cfg`

Reply:
As the Acts PR in the summary says, this is because the config contains loggers in these config structs which are now (v31) unique_ptrs. If we pass by value we need a copy, which aren't allowed. That's why we love them.

Reply:
Thanks! I thought that it was requirement of the interface, but absence of a copy constructor is also a valid reason.